preview-programs/eks-ec2-a1-preview/README.md

```diff
@@ -128,7 +127,7 @@ metadata:
   namespace: kube-system
 data:  
   mapRoles: |  
-    - rolearn: <ARN of instance role (not instance profile) of **linux** node>
+    - rolearn: <ARN of instance role (not instance profile) of **ARM64** nodes from step 6>
```

Comment:
@andyhopp do we need to also have the Linux node role so that we can connect back to the coreDNS that's running there? Something like (per @sramabad1):

```
apiVersion: v1 
kind: ConfigMap 
metadata:  
  name: aws-auth  
  namespace: kube-system 
data:  
  mapRoles: |  
    - rolearn: <ARN of instance role (not instance profile) of **x86s** node> 
      username: system:node:{{EC2PrivateDNSName}} 
      groups: 
        - system:bootstrappers 
        - system:nodes 
    - rolearn: <ARN of instance role (not instance profile) of **ARM64** node> 
      username: system:node:{{EC2PrivateDNSName}} 
      groups: 
        - system:bootstrappers 
        - system:nodes 
```